src/index.html

```diff
@@ -5,8 +5,314 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
     <link rel="stylesheet" href="./styles/main.scss">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a class="logo" href="#home"><img src="images/logo.png" alt="logo"></a>
+
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a
+            class="nav__link is-active"
+            href="#Apple"
+            >Apple</a>
+          </li>
```

Comment:
in all cases
![image](https://github.com/mate-academy/layout_catalog/assets/91411692/0f1546e7-a894-43ac-83c8-72ebe69123b5)
```suggestion
            <a
              class="nav__link is-active"
              href="#Apple"
            >
              Apple
            </a>
          </li>
```